src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/DefaultRazorLanguageServerCustomMessageTarget.cs

```diff
@@ -1051,6 +1052,35 @@ private static TextEdit BuildRevertedEdit(TextEdit provisionalTextEdit)
         return response?.Response;
     }
 
+    public override async Task<string[]?> SimplifyTypeAsync(DelegatedSimplifyTypeNamesParams request, CancellationToken cancellationToken)
+    {
+        var (synchronized, virtualDocument) = await _documentSynchronizer.TrySynchronizeVirtualDocumentAsync<CSharpVirtualDocumentSnapshot>(
+            request.Identifier.Version,
+            request.Identifier.TextDocumentIdentifier.Uri,
+            cancellationToken).ConfigureAwait(false);
```

Comment:
A note for the future, but the pattern for calling `TrySynchronize` in this file has changed a little, if you merge in latest main. Of course, if you merge in latest main you'll break your local roslyn/razor dev experience, so I'm mainly leaving this note here for later :)